looker_deployer/commands/deploy_content.py

```diff
@@ -178,17 +181,17 @@ def deploy_space(s, sdk, env, ini, recursive, debug=False):
     if recursive and space_children:
         logger.info("Attemting Recursion of children folders", extra={"children_folders": space_children})
         for child in space_children:
-            deploy_space(child, sdk, env, ini, recursive, debug)
+            deploy_space(child, sdk, env, ini, recursive, debug, target_base)
     else:
         logger.info("No Recursion specified or empty child list", extra={"children_folders": space_children})
 
 
-def deploy_content(content_type, content, sdk, env, ini, debug=False):
+def deploy_content(content_type, content, sdk, env, ini, debug=False, target_base=None):
     # extract directory path
     dirs = content.rpartition(os.sep)[0] + os.sep
 
     # cut down directory to looker-specific paths
-    a, b, c = dirs.partition("Shared")  # Hard coded to Shared for now TODO: Change this!
+    a, b, c = dirs.partition(target_base)  # Hard coded to Shared for now TODO: Change this!
```

Comment:
The same here, `TODO` should be removed and comment adjusted or removed completely